calicolipidlibrary/MIP2C.py

```diff
@@ -45,40 +45,6 @@ def theoreticalDigest(self):
 			return(FRAGMENTS)
 
 
-		@classmethod
-		def generateLibrary(cls,target=None, mode="pos"):
```

Comment:
This was the first definition of generateLibrary().  It defined a customized `chains` variable.